lib/types.go

```diff
@@ -67,9 +67,10 @@ const (
 
 // Res is a response status, parameters and headers which a test should get
 type Res struct {
-	Status  int
-	Params  map[string]interface{}
-	Headers map[string]string
+	Status      int
+	Params      map[string]interface{}
+	Headers     map[string]string
+	ParamsArray []map[string]interface{}
```

Comment:
Params と ParamsArray の意味合いはほぼ同じなので、隣にあった方が「同じ役割のパラメータなんだな」と分かって親切かなと思いました。

```suggestion
	Params      map[string]interface{}
	ParamsArray []map[string]interface{}
	Headers     map[string]string
```

Reply:
修正しました！